src/6model/serialization.c

```diff
@@ -10,7 +10,7 @@
 
 /* Version of the serialization format that we are currently at and lowest
  * version we support. */
-#define CURRENT_VERSION 19
+#define CURRENT_VERSION 20
```

Comment:
The policy on versioning seems to be: "When making any change that would render a version of MoarVM that doesn't know about a change unable to successfully/completely deserialize things".

Code from before the pull request won't understand the newly added flag and as such fail to deserialize properly.